src/lib/Core.jsx

```diff
@@ -255,9 +255,10 @@ function Core({
             <button
               type="button"
               disabled={answerButtons[index].disabled || false}
-              className={`${answerButtons[index].className} answerBtn btn`}
+              className={`${answerButtons[index].className} answerBtn btn ${(index+1==correctAnswer&&showInstantFeedback)?'correct':''}`}
```

Comment:
lint once

```jsx
className={`${answerButtons[index].className} answerBtn btn ${(index + 1 == correctAnswer && showInstantFeedback) ? 'correct' : ''}`}
```

Reply:
className={`${answerButtons[index].className} answerBtn btn ${(index + 1 === Number(correctAnswer) && showInstantFeedback) ? 'correct' : ''}`}

Added Number(correctAnswer) as I observed from testing locally that correctAnswer was a string, then the check failed

Reply:
@chyke007 Thanks for pointing out. Apart from that, I think there are two cases for `correctAnswer`. 

See

https://github.com/wingkwong/react-quiz-component/blob/04b242b765c040a079bf4f8d70f43bae578bd7ca/src/lib/Core.jsx#L123-L131

Reply:
Thats correct also, that logic wont work for arrays